.github/workflows/format.yml

```diff
@@ -0,0 +1,18 @@
+name: DotNet
+
+on: [pull_request]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    name: Format
+    steps:
+      - uses: actions/checkout@v2
+
+      # workaround for dotnet tools not being on path https://github.com/actions/virtual-environments/issues/213
+      - name: add .dotnet/tools to path
+        run: echo "::set-env name=PATH::${PATH}:${HOME}/.dotnet/tools"
+
+      - run: dotnet tool install -g dotnet-format
+
+      - run: dotnet format --folder ./ --dry-run --check
```

Comment:
This PR will fail if formatting is not done correctly.
See: https://github.com/nmklotas/GitLabApiClient/runs/420703815#step:5:7 for example